src/pdsparsemat.jl

```diff
@@ -57,31 +58,31 @@ LinearAlgebra.sqrt(A::PDSparseMat) = PDMat(sqrt(Hermitian(Matrix(A))))
 
 ### whiten and unwhiten
 
-function whiten!(r::StridedVecOrMat, a::PDSparseMat, x::StridedVecOrMat)
+function whiten!(r::AbstractVecOrMat, a::PDSparseMat, x::AbstractVecOrMat)
     r[:] = sparse(chol_lower(a.chol)) \ x
```

Comment:
Could we use an inplace method instead of creating a copy?

Reply:
I didn't change this code, it is out of scope for the PR